app/controllers/hotels_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HotelsController < ApplicationController
+  def show
+    @hotel = Hotel.find(params[:id])
+    @guests = @hotel.guest_list
+  end
+end
```

Comment:
Similar to my above comment, you can call `guest_list` directly on the Hotel object in your view. So, no need to pass over a new variable.